application/controllers/admin/Statistics.php

```diff
@@ -669,7 +669,7 @@ public function simpleStatistics($surveyid)
 
         if (!$oSurvey->isActive) {
             Yii::app()->setFlashMessage(gT("This survey is not active and has no responses."), 'error');
-            $this->getController()->redirect($this->getController()->createUrl("/surveyAdministration/view/surveyid/{$oSurveyid}"));
+            $this->getController()->redirect($this->getController()->createUrl("/surveyAdministration/view/surveyid/{$surveyid}"));
```

Comment:
this should probably use the sanitized variable iSurveyId

Reply:
Agree